src/package/packageVersionCreate.ts

```diff
@@ -91,71 +91,15 @@ export class PackageVersionCreate {
     }
   }
 
-  private async validateDependencyValues(dependency: PackageDescriptorJson): Promise<void> {
-    // If valid 04t package, just return it to be used straight away.
-    if (dependency.subscriberPackageVersionId) {
-      pkgUtils.validateId(pkgUtils.BY_LABEL.SUBSCRIBER_PACKAGE_VERSION_ID, dependency.subscriberPackageVersionId);
-      return;
-    }
-
-    if (dependency.packageId && dependency.package) {
-      throw messages.createError('errorPackageAndPackageIdCollision', []);
-    }
-
-    const idOrPackage = dependency.packageId ?? dependency.package;
-    if (!idOrPackage) {
-      throw messages.createError('errorPackageOrPackageIdMissing', []);
-    }
-
-    const packageIdFromAlias = this.project.getPackageIdFromAlias(idOrPackage) ?? idOrPackage;
-
-    // If valid 04t package, just return it to be used straight away.
-    if (pkgUtils.validateIdNoThrow(pkgUtils.BY_LABEL.SUBSCRIBER_PACKAGE_VERSION_ID, packageIdFromAlias)) {
-      dependency.subscriberPackageVersionId = packageIdFromAlias;
-      return;
-    }
-
-    if (!packageIdFromAlias || !dependency.versionNumber) {
-      throw messages.createError('errorDependencyPair', [JSON.stringify(dependency)]);
-    }
-
-    // Just override dependency.packageId value to the resolved alias.
-    dependency.packageId = packageIdFromAlias;
-
-    pkgUtils.validateId(pkgUtils.BY_LABEL.PACKAGE_ID, dependency.packageId);
-    validateVersionNumber(
-      dependency.versionNumber,
-      BuildNumberToken.LATEST_BUILD_NUMBER_TOKEN,
-      BuildNumberToken.RELEASED_BUILD_NUMBER_TOKEN
-    );
-    await this.validatePatchVersion(dependency.versionNumber, dependency.packageId);
-
-    // Validate that the Package2 id exists on the server
-    const query = `SELECT Id FROM Package2 WHERE Id = '${dependency.packageId}'`;
-    const result = await this.connection.tooling.query<{ Id: string }>(query);
-
-    if (!result.records || result.records.length !== 1) {
-      throw messages.createError('errorNoIdInHub', [dependency.packageId]);
-    }
-  }
-
   /**
-   * A dependency in the workspace config file may be specified using either a subscriber package version id (04t)
+   * A dependency in the project config file may be specified using either a subscriber package version id (04t)
    * or a package Id (0Ho) + a version number.  Additionally, a build number may be the actual build number, or a
    * keyword: LATEST or RELEASED (meaning the latest or released build number for a given major.minor.patch).
    *
    * This method resolves a package Id + version number to a subscriber package version id (04t)
    * and adds it as a SubscriberPackageVersionId parameter in the dependency object.
    */
-  private async retrieveSubscriberPackageVersionId(dependency: PackageDescriptorJson): Promise<PackageDescriptorJson> {
-    await this.validateDependencyValues(dependency);
-    if (dependency.subscriberPackageVersionId) {
```

Comment:
do we need to call `resolveSubscriberPackageVersionId` here now since we moved this into its own method?